File: Q02/main.c
#include<stdio.h>
int main(){
    int row,col;
    scanf("%d%d",&row,&col);

    int arr[row][col],sum=0;

    for(int i=0;i<row;i++){
        for(int j=0;j<col;j++){
            scanf("%d",&arr[i][j]);
            sum+=arr[i][j];
        }
    }
    printf("%d",sum);
    return 0;
}

File: Q02/main.py
row = int(input())
col = int(input())

arr=[]
sum=0

for i in range(row):
    arr.append(list(map(int,input().split()))[:col])

for i in range(row):
    for j in range(col):
        sum+=arr[i][j]
print(sum)